src/core/congestion_control.c

```diff
@@ -434,7 +434,7 @@ QuicCongestionControlOnDataAcknowledged(
         // DeltaT < ~2.5M (about 30min).
         //
 
-        int64_t DeltaT = TimeInCongAvoid - Cc->KCubic + US_TO_MS(SmoothedRtt);
+        int64_t DeltaT = TimeInCongAvoid - (uint64_t)Cc->KCubic + (uint64_t)US_TO_MS(SmoothedRtt);
```

Comment:
Actually, I'm not sure this should ever be negative....

Reply:
Supposedly CubicWindow below could be negative.

Reply:
@maolson-msft would you mind commenting on this logic? MacOS is complaining about integer overflow. I think just casting all these to `int64_t` before the arithmetic would suffice. Thoughts?